Logger/Logger.php

```diff
@@ -37,7 +37,7 @@ public function __construct(
      * @param array $context
      * @return bool
      */
-    public function addRecord(int $level, string $message, array $context = []): bool
+    public function addRecord($level, $message, array $context = [])
```

Comment:
Aqui está sendo modificado as assinaturas do método, porém nesse PR https://github.com/melhorenvio/magento2/pull/11 esse método é removido, está correto isso? Se for o mesmo removido no outro PR, podemos dar decline nesse PR?